distributed/protocol/tests/test_polars.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+import pytest
+
+pl = pytest.importorskip("polars")
+
+import distributed
+from distributed.protocol import deserialize, serialize, to_serialize
+from distributed.utils_test import gen_cluster
+
+
+def test_roundtrip():
+    # Test that the serialize/deserialize functions actually
+    # work independent of distributed
+    obj = pl.DataFrame({"A": list("abc"), "B": [1, 2, 3]})
+    header, frames = serialize(obj)
+    new_obj = deserialize(header, frames)
+    assert obj.frame_equal(new_obj)
+
+
+def test_roundtrip_lazy():
+    # Test that the serialize/deserialize functions actually
+    # work independent of distributed
+    obj = pl.DataFrame({"A": list("abc"), "B": [1, 2, 3]}).lazy()
+    header, frames = serialize(obj)
+    new_obj = deserialize(header, frames)
+    assert obj.collect().frame_equal(new_obj.collect())
+
+
+def echo(arg):
+    return arg
+
+
+def test_scatter():
+    @gen_cluster(client=True)
+    async def run_test(client, scheduler, worker1, worker2):
+        obj = pl.DataFrame({"A": list("abc"), "B": [1, 2, 3]})
+        obj_fut = await client.scatter(obj)
+        fut = client.submit(echo, obj_fut)
+        result = await fut
+        assert obj.frame_equal(result)
+
+    run_test()
```

Comment:
You shouldn't need the `test_scatter` function here.  You should be able to rename `run_test` as `test_scatter` and put it on the top level.

Also, by convention we do ...

```python
@gen_cluster(client=True)
async def test_foo(c, s, a, b):
    ...
```

So I recommend removing the longer names client, scheduler, worker1, worker2.  They make more sense, yes, but when you have thousands of these it's nicer to use the shorter names.